modules/exploits/windows/local/cve_2021_40449.rb

```diff
@@ -0,0 +1,164 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = GoodRanking
+
+  include Msf::Post::File
+  include Msf::Post::Windows::Priv
+  include Msf::Post::Windows::Process
+  include Msf::Post::Windows::ReflectiveDLLInjection
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        {
+          'Name' => '',
```

Comment:
'Name' => 'Microsoft Windows CallbackHell CVE-2021-40449 Win32k Use After Free Local Privilege Elevation', or similar? :trollface:

Reply:
Can I not just name it `Nameless Exploit 202`? :trollface: Just kidding I'll fix this up 😄